### ISSUE TITLE
 I want to know where the epoch_num are set

### ISSUE THREAD

**Issue by Ostnie** [nilboy/tensorflow-yolo#39]
I have run the demo successfully,but 24 hours later,it's still running,and I am using 1080ti,Losses are always fluctuating between 5 and 10,I don't know whether the train has been finished,so I want to find the epoch_num,but I can't find it,anyone konw this?many thanks!

**Comment by SurajDonthi** [comment]
> I have run the demo successfully,but 24 hours later,it's still running,and I am using 1080ti,Losses are always fluctuating between 5 and 10,I don't know whether the train has been finished,so I want to find the epoch_num,but I can't find it,anyone konw this?many thanks!

You can change `max_iterations` in conf/train.cfg. It is currently set at 1000000. To counter this change, you can just increase the learning rate and try it out.
